tests/common/models.py

```diff
@@ -205,20 +205,17 @@ class SimpleWeightTiedModel(ComposerClassifier):
     def __init__(self, num_features: int = 1, device: str = 'cpu') -> None:
         self.num_features = num_features
 
-        mlp = SimpleMLP(num_features, device)
-
+        fc1 = torch.nn.Linear(num_features, num_features, device=device, bias=False)
```

Comment:
I had to remove the redundant modules since this keeps throwing errors. Will look into this in more detail tomorrow though